libs/src/services/creatorNode/index.js

```diff
@@ -705,11 +716,27 @@ class CreatorNode {
     }
     headers['X-Session-ID'] = this.authToken
 
-    const requestId = uuid()
+    const requestId = getUuid()
```

Comment:
why uuid4? can we change it elsewhere so it's consistent?

Reply:
the reason went with uuidv4 was because for some reason, the uuid() method wasn't working at the time of testing. can test again and see if there are any existing issues

Reply:
tested with original uuid method and works. reverting